frontend/javascripts/oxalis/model/reducers/volumetracing_reducer.ts

```diff
@@ -194,6 +198,33 @@ function handleUpdateSegment(state: OxalisState, action: UpdateSegmentAction) {
   });
 }
 
+function handleFocusSegment(state: OxalisState, action: ClickSegmentAction) {
```

Comment:
I'd rename this method to `expandSegmentParents` to make it clearer what is exactly happening.